network-resources-injector/values.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022 K8s Network Plumbing Group
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+deployment:
+  replicas: 2 
+
+serviceAccount:
+  name: network-resources-injector-sa
+  
+secret:
+  name: network-resources-injector-sa-secret
+  
+role:
+  secretsName: network-resources-injector-secrets
+  webhookconfigName: network-resources-injector-webhook-configs
+  serviceName: network-resources-injector-service
+  configmapsName: network-resources-injector-configmaps
+  
+roleBinding:
+  name: network-resources-injector-role-binding
+  secretsName: network-resources-injector-secrets-role-binding
+  webhookconfigName: network-resources-injector-webhook-configs-role-binding
+  serviceName: network-resources-injector-service-role-binding
+  configmapsName: network-resources-injector-configmaps-role-binding
+    
+image:
+  repository: network-resources-injector
+  tag: latest
+  pullPolicy: IfNotPresent
+
+service:
+  name: network-resources-injector-service
+  port: 443
+  targetPort: 8443
+
+webhookconf:
```

Comment:
nit: webhookconf => webhook?

Reply:
There is webhook as key in child hierarchy also, so this was to distinguish the webhook config(kind) to the actual webhook(one in webhooks in the webhookconfig). I can change it if you feel this is better.